pkg/routeagent_driver/handlers/kubeproxy/endpoint_handler.go

```diff
@@ -105,7 +105,7 @@ func (kp *SyncHandler) RemoteEndpointCreated(endpoint *submV1.Endpoint) error {
 	kp.syncHandlerMutex.Lock()
 	defer kp.syncHandlerMutex.Unlock()
 
-	lastProcessedTime, ok := kp.remoteEndpointTimeStamp[endpoint.ClusterName]
+	lastProcessedTime, ok := kp.remoteEndpointTimeStamp[endpoint.Name]
```

Comment:
The original issue fixed [157 ]( https://github.com/submariner-io/submariner/pull/1571) was for two different endpoints from the same cluster. As you mentioned you could try with endpoint.Spec.ClusterId

Reply:
Done